backend/python/app/rest/entity_routes.py

```diff
@@ -135,3 +154,16 @@ def delete_entity(id):
         return jsonify({"error": (error_message if error_message else str(e))}), 500
 
     return jsonify(result), 200
+
+
+# defines GET endpoint for a URL to the entity's file with the provided uuid
+@blueprint.route("/files/<string:id>", methods=["GET"], strict_slashes=False)
+@require_authorization_by_role({"user", "Admin"})
+def get_file(id):
+    try:
+        file_url = FileStorageService.get_file(id)
```

Comment:
@zm2he `get_file` is not a static method

Reply:
Made file storage service an instance variable in routes before passing it to entity service. Will test to see if it works.